### PR TITLE
Create trellix-endpoint-security.json

### PR DESCRIPTION
**Pull request by mikiodehartj1**
initial build

### Alerts Rule Packs
- [ ] Does a related alert already exist, and should it be tweaked or added to instead?
- [ ] Test each query to make sure it works
- [ ] Look for hardcoded variables/parameter values in the query
- [ ] Consider Severity for Alerts
- [ ] Spellcheck
- [ ] Use all caps for J1QL keywords and relationship classes
- [ ] Upload the alerts rule pack JSON into JupiterOne to validate

### CODE REVIEW

File: rule-packs/trellix-endpoint-security.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
+[
+    {
+        "name": "trellix-threats-1",
+        "description": "This query will return threats with an unresolved status",
+        "queries": [
+            {
+                "name": "query0",
+                "query": "FIND trellix_threat with status = unresolved",
```
Comment (SeaBlooms) [inline rule-packs/trellix-endpoint-security.json:8]: can we make all of the J1QL keywords in caps?

File: rule-packs/trellix-endpoint-security.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        ],
+        "alertLevel": "INFO"
+    },
+    {
+        "name": "trellix-threats-4",
+        "description": "This will alert when a device is First Seen.",
+        "queries": [
+            {
+                "name": "query0",
+                "query": "FIND trellix_device with createdOn/firstSeenOn < 24 hours return device.name, device.user ",
```
Comment (SeaBlooms) [inline rule-packs/trellix-endpoint-security.json:80]: gotta fix up this query, remove the forward slash

**Review by SeaBlooms — CHANGES_REQUESTED** [review]
need to update index file to define new rule pack per the merges test fails - also a couple updates to the queries

**Review by mikiodehartj1 — COMMENTED** [review]
update syntax and index

**Review by mikiodehartj1 — COMMENTED** [review]
updates to ql syntax

File: rule-packs/trellix-endpoint-security.json (hunk 1 above)
Comment (mikiodehartj1) [reply rule-packs/trellix-endpoint-security.json]: Done

File: rule-packs/trellix-endpoint-security.json (hunk 2 above)
Comment (mikiodehartj1) [reply rule-packs/trellix-endpoint-security.json]: Done

**Pull request closed without merging**